designer/client/src/components/sidePanels/SearchLabeledInput.tsx

```diff
@@ -0,0 +1,19 @@
+import { forwardRef, PropsWithChildren } from "react";
+import { FormControl } from "@mui/material";
+import { nodeInput } from "../graph/node-modal/NodeDetailsContent/NodeTableStyled";
+import React from "react";
+
+export type SearchLabeledInputProps = PropsWithChildren<{
+    name: string;
+}>;
+
+export const SearchLabeledInput = forwardRef<HTMLInputElement, SearchLabeledInputProps>(({ children, ...props }, ref) => {
```

Comment:
Can we adjust the style, increase the elements' width, and add padding 16px? You can use `theme.spacing`, like [here](https://github.com/TouK/nussknacker/blob/90a4c5020a7df2568ce9ebb5aa96cb619eaf2c77/designer/client/src/components/tips/Styled.tsx#L32) 
<img width="452" alt="image" src="https://github.com/user-attachments/assets/15340131-77f7-4114-98f3-682e848f6e98">